tests/unit/api/test_job_api.py

```diff
@@ -70,11 +70,11 @@
 def test_get_jobs(flask_app, mocker: MockFixture):
     get_jobs = mocker.patch.object(job_db, "get_jobs", return_value=JOB_LIST)
     response = flask_app.get(
-        url_for(
-            "job_api.get_jobs",
-            status="completed",
-            operation=["ADD", "CHANGE", "PATCH_METADATA"],
-        ),
+        "/jobs",
+        json={
```

Comment:
is this supposed to be sent as json? 🤔 I think this endpoint expect query params?

Reply:
Good catch!